src/items.rs

```diff
@@ -630,7 +630,7 @@ impl<'a> FmtVisitor<'a> {
             items = itemize_list_with(0);
         }
 
-        let shape = self.shape().sub_width(2)?;
+        let shape = self.shape();
```

Comment:
FYI @ytmimi, this `.sub_width(2)?` has no effect on the tests that are currently in the repo passing or faililng. 

I chose to remove it as less code is better? Thought I would flag as you probably have more insight into what this might affect and I'm happy to write a test case to document it too 😄

Reply:
Please revert this change as I don't think it's related to the bug we're trying to solve for. overall the `Shape` dictates how far we're currently indented, and how much available space rustfmt has to work with in order to rewrite the current AST node on one line. There's a doc string on the `Shape` struct that you can read for more details.

We'll usually call `sub_width(n)` to reduce the width for some reason. In this case it's unclear why we're reducing the width by 2 but I wouldn't mess with this if we don't have to.

Reply:
Thank you so much for the brief overview, that seems to be inline with what I was thinking. I apprechiate you taking the time to explain that to me.

I've added this line back in.